samples/wifi/wfa_qt_app/README.rst

```diff
@@ -12,24 +12,56 @@ This sample demonstrates how to use the WFA QuickTrack (WFA QT) library needed f
 Requirements
 ************
 
-The sample supports the following development kit:
+The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
 Overview
 ********
 
-This sample acts as a control application for the WFA QT library, to be used for Wi-Fi Alliance QuickTrack certification.
+Nordic's QuickTrack sample and library offer support for QuickTrack certification testing through two distinct interfaces: a netUSB interface and a SLIP interface (Serial Line Internet Protocol). We can choose either of these options for running QuickTrack certification.
+
 See `Wi-Fi Alliance Certification for nRF70 Series <Wi-Fi Alliance Certification for nRF70 Series_>`_ for more information.
 
+Test setup
+==========
+
+The following figure shows a reference test setup.
+
+.. figure:: /images/wifi_quicktrack_test_setup.svg
+     :alt: Wi-Fi QuickTrack test setup
+
+     Wi-Fi QuickTrack reference test setup
+
+
+Build configuration
+*******************
+
+|config|
+
+Configuration options
+=====================
+
+The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/wfa_qt_app/Kconfig`) :
```

Comment:
It seems there is only one option.
```suggestion
The following sample-specific Kconfig option is used in this sample (located in :file:`samples/wifi/wfa_qt_app/Kconfig`) :
```

Reply:
> It seems there is only one option.

Mutliple options are added, so not addressing this.